pyprocessing/color.py

```diff
@@ -4,25 +4,40 @@
 
 
 class Color:
-    RGB = 0
-    HSV = 1
+    RGB = 1
     HLS = 2
+    HSB = 3
 
-    def __init__(self, *args, colorspace=0):
-        self.colorspace = colorspace
+    colorspace = RGB
+    maxv1 = maxv2 = maxv3 = maxva = 255
+
+    def __init__(self, *args):
+        def adjust(val, max):
+            if val > max:
+                return 255
+            return round((val / max) * 255)
+
+        def adjusttuple(tup, maxes):
+            return (adjust(i, j) for i, j in zip(tup, maxes))
 
         if len(args) == 1:
             # 1 argument : no matter the colorspace, it's grayscale
-            self.red = self.green = self.blue = args[0]
+            self.red = adjust(args[0], self.maxv1)
+            self.green = adjust(args[0], self.maxv2)
+            self.blue = adjust(args[0], self.maxv3)
             self.alpha = 255
         elif len(args) == 2:
-            self.red = self.green = self.blue = args[0]
-            self.alpha = args[1]
+            self.red = adjust(args[0], self.maxv1)
+            self.green = adjust(args[0], self.maxv2)
+            self.blue = adjust(args[0], self.maxv3)
+            self.alpha = adjust(args[1], self.maxva)
         elif len(args) == 3:
             self.alpha = 255
-            self.red, self.green, self.blue = self._values_to_rgb(*args)
+            self.red, self.green, self.blue = self._values_to_rgb(*adjusttuple(  # Adjust vals then unpack and convert
+                args, (self.maxv1, self.maxv2, self.maxv3)                       # Terribly done by Peanutbutter_Warrior
+            ))
```

Comment:
formatting. Comment should be above. Don't credit yourself in those comments (contributions should be listed in the AUTHORS.md file)